src/H5FDpublic.h

```diff
@@ -185,6 +193,9 @@
 #define H5FD_CTL__GET_MPI_COMMUNICATOR_OPCODE 2
 #define H5FD_CTL__GET_MPI_RANK_OPCODE         3
 #define H5FD_CTL__GET_MPI_SIZE_OPCODE         4
+#define H5FD_CTL__MEM_ALLOC                   5
+#define H5FD_CTL__MEM_FREE                    6
+#define H5FD_CTL__MEM_COPY                    7
```

Comment:
Here, I only reserved the op codes for driver-level memory allocations and frees. I haven't added any argument structs for them since there isn't a use case yet, but can do so if it's useful.

Reply:
Agree -- wait until we have use cases.